app/screens/channel/channel_nav_bar/channel_title/channel_title.js

```diff
@@ -43,9 +45,31 @@ export default class ChannelTitle extends PureComponent {
     }
 
     render() {
-        const {currentChannelName, displayName, isChannelMuted, onPress, theme} = this.props;
-        const channelName = displayName || currentChannelName;
+        const {currentChannelName, displayName, isChannelMuted, onPress, theme, guests} = this.props;
+
         const style = getStyle(theme);
+
+        let channelName = displayName || currentChannelName;
+        if (guests) {
+            const users = channelName.split(',');
+            const isGuest = guests.split(',').map((v) => v === 'GUEST');
+
+            channelName = [];
+            users.forEach((value, index) => {
```

Comment:
This is how I add the tags to the channel title, which is the weirdest part of this PR. Probably is a better way to solve it. Besides this doesn't respect properly the GuestTag styles (border radius, padding...).